app/views/abouts/show.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class="font-karla flex flex-col items-center w-full">
+    <div class="bg-blue-marine flex flex-col items-center px-5 pt-6 pb-10 w-full">
+        <div class="max-w-screen-sm text-center font-normal">
+            <p class="text-base mb-6 text-white">Brought to you by</p>
+            <%= inline_svg_tag "telos_labs_logo.svg", class: "mb-10 w-full" %>
+            <p class="text-white text-lg mb-6">We designed and developed this open-source app for the Rails community, you can contribute to the repo <a href="https://github.com/TelosLabs/rails-world" target="_blank" class="text-blue-teal">here</a>.</p>
+            <p class="text-white text-lg mb-10">Telos creates disruptive, purpose-driven software for visionary clients to enhance productivity and enrich lives.</p>
+            <div class="flex flex-row w-full items-center gap-5">
+                <a href="https://hi.teloslabs.co/" target="_blank" class="grow bg-blue-teal rounded-[10px] p-4 flex flex-row items-center justify-center">
+                    <%= inline_svg_tag "icon_web.svg", class: "w-6 h-6" %>
+                </a>
+                <a href="https://www.linkedin.com/company/telos-labs" target="_blank" class="grow bg-blue-teal rounded-[10px] p-4 flex flex-row items-center justify-center">
+                    <%= inline_svg_tag "icon_linkedin.svg", class: "w-6 h-6" %>
+                </a>
+            </div>
+        </div>
+    </div>
```

Comment:
We're missing the report a bug section. Should it be included in this PR?
![image](https://github.com/user-attachments/assets/101e491b-0a60-4db8-beac-4fa61951b4ec)

Reply:
The ticket mentioned we shouldn't include it on this PR